src/styles/_table.scss

```diff
@@ -1,4 +1,4 @@
-#table-container {
+.table-container {
   max-height: 95vh;
   overflow: scroll;
 }
```

Comment:
Scroll bar seems to be missing from the table and it displays all of the logs now

Reply:
Whoops, updated the wrong stylesheet. Should be fixed now.